components/content/ProseA.vue

```diff
@@ -8,7 +8,7 @@
     <span v-if="isExternalLink || isApiReferenceLink">&nbsp;</span>
     <span
       v-if="isApiReferenceLink"
-      class="rounded bg-interactive-quaternary h-5 w-10 inline-flex justify-center items-center"
+      class="rounded-sm bg-interactive-quaternary h-3 w-6 inline-flex justify-center items-center"
```

Comment:
Instead of a fixed height and width you can just remove this and it looks quite good. IMO this doesn't have enough spacing between the text and the top.

Reply:
@MeganSteenkamp I've made the text smaller which has increased this gap - have a look at the updated screenshot above. Do you think the gap should be even bigger?